src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -5,26 +5,88 @@
             [quo2.components.tags.context-tags :as context-tags]
             [quo2.foundations.colors :as colors]
             [reagent.core :as reagent]
+            [status-im.constants :as constants]
             [status-im.i18n.i18n :as i18n]
+            [status-im.multiaccounts.core :as multiaccounts]
             [status-im.ui.components.topbar :as topbar]
+            [status-im.utils.datetime :as datetime]
             [status-im.utils.handlers :refer [<sub >evt]]))
 
+(defn activity-title
+  [{:keys [type]}]
+  (case type
+    constants/activity-center-notification-type-contact-request
+    (i18n/label :t/contact-request)
+
+    constants/activity-center-notification-type-one-to-one-chat
+    "Dummy 1:1 chat title"
+
+    "Dummy default title"))
```

Comment:
FYI: More cases for different notification types will be covered as the new Activity Center issues are played. This is true for this function and for others in this namespace.